.changeset/fifty-apples-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@builder.io/sdk": patch
```

Comment:
`@builder.io/sdk` is the package containing core logic of gen1 SDK. What you want is to make a changeset for every SDK with the name `@builder.io/sdk-*`